src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -51,7 +51,10 @@ impl Diff {
     /// Specify the actual output for the diff from a file.
     pub fn actual_file<P: AsRef<Path>>(&mut self, path: P) -> &mut Self {
         let path = path.as_ref();
-        let content = std::fs::read_to_string(path).expect("failed to read file");
+        let content = match std::fs::read_to_string(path) {
+            Ok(c) => c,
+            Err(e) => panic!("Failed to read `{}`: {:?}", path.display(), e),
```

Comment:
Nit (very not important): there really isn't much consistency or style guide but I think the ported tests usually use lower-case error messages just like rustc

Reply:
Updated it. Better to have consistent error messages.